android/app-example/src/main/java/com/badoo/ribs/example/rib/dialog_example/DialogExampleInteractor.kt

```diff
@@ -0,0 +1,77 @@
+package com.badoo.ribs.example.rib.dialog_example
+
+import android.arch.lifecycle.Lifecycle
+import com.badoo.mvicore.android.lifecycle.createDestroy
+import com.badoo.ribs.core.Interactor
+import com.badoo.ribs.core.Router
+import com.badoo.ribs.dialog.Dialog
+import com.badoo.ribs.example.rib.dialog_example.DialogExampleRouter.Configuration
+import com.badoo.ribs.example.rib.dialog_example.DialogExampleView.Event.ShowRibDialogClicked
+import com.badoo.ribs.example.rib.dialog_example.DialogExampleView.Event.ShowSimpleDialogClicked
+import com.badoo.ribs.example.rib.dialog_example.DialogExampleView.ViewModel
+import com.badoo.ribs.example.rib.dialog_example.dialog.RibDialog
+import com.badoo.ribs.example.rib.dialog_example.dialog.SimpleDialog
+import com.badoo.ribs.example.rib.lorem_ipsum.LoremIpsum
+import com.jakewharton.rxrelay2.BehaviorRelay
+import io.reactivex.functions.Consumer
+
+class DialogExampleInteractor(
```

Comment:
The whole rib is named DialogExample, everything else is just the usual suffix. I think it's pretty self-explanatory, but feel free to suggest otherwise.